generated_code/generate.py

```diff
@@ -176,11 +180,50 @@
   except:
     print('WARNING: ChainForge was not found. Falling back to GemmForge.')
 
+trueOutputDir = os.path.join(cmdLineArgs.outputDir, 'equation')
+if not os.path.exists(trueOutputDir):
+  os.mkdir(trueOutputDir)
+
+subfolders += ['generated_code/equation']
 
 # Generate code
 gemmTools = GeneratorCollection(gemm_generators)
-generator.generate(outputDir=cmdLineArgs.outputDir,
+generator.generate(outputDir=trueOutputDir,
                    namespace='seissol',
                    gemm_cfg=gemmTools,
                    cost_estimator=cost_estimators,
                    include_tensors=include_tensors)
+
+def generate_general(subfolders):
+  arch = useArchitectureIdentifiedBy('d' + cmdLineArgs.host_arch[1:])
```

Comment:
What about single precision here?

Reply:
That part was more meant to be precision-agnostic—i.e. the stiffness tensors etc.

(originally, it came from #932, to try to support multi-precision as well... Then forcing `double` was a good way to avoid templating in these classes as well)

Reply:
Makes sense. Can you add a comment that you use `double` here, since this kernel is not used at runtime?

Reply:
Done. In the same vein, couldn't we probably keep the anisotropy-related kernels; especially when going towards multi-material anyways?